code/midtown/mmaudio/sound.h

```diff
@@ -275,7 +277,7 @@ class AudSound final : public asNode
     // ?GetSoftFreqChange2DFlags@AudSound@@SA?BKXZ | unused
     ARTS_IMPORT static const ulong GetSoftFreqChange2DFlags();
 
-private:
+public:
```

Comment:
I'd rather leave this as private, then add a `public` specifically for the member variables